.github/workflows/deploy.frontend-webapp.yml

```diff
@@ -0,0 +1,35 @@
+name: Build and Deploy Frontend WebApp

```

Comment:
Generally in the right direction, so only a couple of notes - but there's a general problem of the frontend has been deployed to UAT and now throws an error, so something isn't right.